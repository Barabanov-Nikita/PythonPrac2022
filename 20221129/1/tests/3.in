class C(metaclass=dump):
    def __init__(self, val):
        self.val = self.seven(val)

    def seven(self, val, **kwargs):
        return val + 7


c = C(10)
print(c.seven(7))
print(c.seven(0, another=42))