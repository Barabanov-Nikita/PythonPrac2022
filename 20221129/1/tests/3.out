__init__: (10,), {}
seven: (10,), {}
seven: (7,), {}
14
seven: (0,), {'another': 42}
7
